packages/api/test/unit/beacon/oapiSpec.test.ts

```diff
@@ -84,11 +84,105 @@ const testDatas = {
   ...validatorTestData,
 };
 
+const filteredOperations = [
```

Comment:
Here we use term "filtered" but below we use "ignored" (also have ignoredTopics). For me "ignored" makes it a bit clearer

Reply:
Makes sense